examples/rf.py

```diff
@@ -89,7 +76,7 @@ def rfr(cfg, seed):
 # Or we can add multiple hyperparameters at once:
 num_trees = UniformIntegerHyperparameter("num_trees", 10, 50, default=10)
 max_depth = UniformIntegerHyperparameter("max_depth", 20, 30, default=20)
```

Comment:
Hm, maximum depth of a random forest is normally not tuned if this isn't a problem with memory.